usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -62,14 +63,24 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
         )
     )
 
+    scap_report = os.environ.get("SCAP_REPORT", "")
+    if scap_report:
+        scap_report = "--report {}".format(scap_report)
+
     result = host.run(
-        "sudo oscap xccdf eval --local-files {dir} --profile {profile} {file}".format(
+        "sudo oscap xccdf eval {scap_report} --local-files {dir} --profile {profile} {file}".format(
```

Comment:
shouldn't here be used `report` as variable, in place of scap_report ?

Reply:
> shouldn't here be used `report` as variable, in place of scap_report ?

Yes. Good catch!